index.html

```diff
@@ -918,6 +937,37 @@ <h1 class="editableArea">Headline1</h1>
 						</div>
 					</div>
 				</li>
+
+				<li title="arrow">
+					<div class="newMockElement boxarrow">
```

Comment:
Would it be possible to have one element (`div`) in one `li`? – that would be consistent with the rest of the MockElements